react-components/src/architecture/base/utilities/IconName.ts

```diff
@@ -0,0 +1,39 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+
+export type IconName =
```

Comment:
This don't scale. Assume you defined a command outside react-component, and you are stuck